plugins/statistics.koplugin/main.lua

```diff
@@ -130,6 +130,7 @@ ReaderStatistics.default_settings = {
 }
 
 function ReaderStatistics:onDispatcherRegisterActions()
+    Dispatcher:registerAction("stats_calendar_day_view", {category="none", event="ShowCalendarDayView", title=_("Open today's statistics timeline"), general=true, separator=false})
```

Comment:
I spend a few moments looking for it. `Open today's statistics timeline` wasn't catching my eyes as I expected Calendar or statistics as the first word. Also, "Open" is not needed. May be just:
`Statistics today's timeline` ?
Also dunno about the order, most recent added after older ones? Or by increasing granularity?
```
Statistics calendar view
Statistics today's timeline
```